# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/libreoffice/hotfix-no-destdir-grep.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- libreoffice-24.2.3.2/bin/distro-install-clean-up.vanilla	2024-05-29 09:54:10.267506353 +0200
+++ libreoffice-24.2.3.2/bin/distro-install-clean-up	2024-05-29 09:55:58.332495940 +0200
@@ -72,7 +72,7 @@
     fi
 done
 
-if test "z$DESTDIR" != "z" ; then
+if false; then
     echo "Checking for DESTDIR inside installed files..."
     found_destdir=
     for file in `find $DESTDIR -type f` ; do
